jovian/jovian-jupyter-lab-extension/Publishing-Guide.md

```diff
@@ -0,0 +1,70 @@
+## Guide To Publish the Extension to npm
+---
+
+### Before you begin register for a [npm](https://www.npmjs.com/ "npm") account and enable 2FA, and login through the console:
+
+```shell
+$ npm login
+```
+
+---
+### In order to publish the package, you need to have ``` package.json``` file in the directry for npm to create a package.
```

Comment:
@saboomayank can you create section under https://github.com/JovianML/jovian-py/blob/master/CONTRIBUTING.md add about publishing there. 
@bendesu you write about local dev environment of jupyter lab.
**In a separate PR**